modules/serverless-jaeger-config.adoc

```diff
@@ -56,7 +56,7 @@ spec:
       debug: "false" <4>
 ----
 +
-<1> The `sample-rate` defines sampling probability. Using `sample-rate: "0.1"` means that 1 in 10 traces will be sampled.
+<1> The `sample-rate` defines sampling probability. Using `sample-rate: "0.1"` means that 1 in 10 traces are sampled.
 <2> `backend` must be set to `zipkin`.
 <3> The `zipkin-endpoint` must point to your `jaeger-collector` service endpoint. To get this endpoint, substitute the namespace where the Jaeger CR is applied.
 <4> Debugging should be set to `false`. Enabling debug mode by setting `debug: "true"` allows all spans to be sent to the server, bypassing sampling.
```

Comment:
This file also has this below:
```
oc get route jaeger
```
I think it would be good to mention the namespace so that the command is complete. The user might have changed namespaces in the meantime and the command would not work for them. Let's use this:
```
oc get route jaeger -n default
```